src/proxy-ui-api/frontend/src/views/Settings/SystemParameters/SystemParameters.vue

```diff
@@ -0,0 +1,175 @@
+              </tr>
+              </thead>
+              <tbody data-test="system-parameters-approved-ca-table-body">
+              <tr v-for="approvedCA in orderedCertificateAuthorities" :key="approvedCA.path">
+                <td :class="{'interm-ca': !approvedCA.top_ca}">{{approvedCA.subject_distinguished_name}}</td>
+                <td v-if="approvedCA.top_ca">{{$t('systemParameters.approvedCertificateAuthorities.table.ocspResponse.NOT_AVAILABLE')}}</td>
+                <td v-if="!approvedCA.top_ca">{{$t(`systemParameters.approvedCertificateAuthorities.table.ocspResponse.${approvedCA.ocsp_response}`)}}</td>
+                <td>{{approvedCA.not_after | formatDate}}</td>
+              </tr>
+              </tbody>
+            </table>
+          </v-col>
+        </v-row>
+      </v-container>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import LargeButton from '@/components/ui/LargeButton.vue';
+import SmallButton from '@/components/ui/SmallButton.vue';
+import { Anchor, CertificateAuthority, TimestampingService } from '@/types';
+import * as api from '@/util/api';
+import { Permissions } from '@/global';
+
+export default Vue.extend({
+  components: {
+    LargeButton,
+    SmallButton,
+  },
+  data() {
+    return {
+      configuratonAnchor: {} as Anchor,
+      configuredTimestampingServices: [] as TimestampingService[],
+      certificateAuthorities: [] as CertificateAuthority[],
+      permissions: Permissions,
+    };
+  },
+  computed: {
+    orderedCertificateAuthorities(): CertificateAuthority[] {
+      return this.certificateAuthorities.sort((authorityA, authorityB) => authorityA.path.localeCompare(authorityB.path));
+    },
+  },
+  methods: {
+    hasPermission(permission: Permissions): boolean {
+      return this.$store.getters.hasPermission(permission);
+    },
+    async fetchConfigurationAnchor() {
+      return api.get('/system/anchor')
+        .then((resp) => this.configuratonAnchor = resp.data)
+        .catch((error) => this.$bus.$emit('show-error', error.message));
+    },
+    async fetchConfiguredTimestampingServiced() {
+      return api.get('/system/timestamping-services')
+        .then((resp) => this.configuredTimestampingServices = resp.data)
+        .catch((error) => this.$bus.$emit('show-error', error.message));
+    },
+    async fetchApprovedCertificateAuthorities() {
+      return api.get('/certificate-authorities?include_intermediate_cas=true')
+        .then((resp) => this.certificateAuthorities = resp.data)
+        .catch((error) => this.$bus.$emit('show-error', error.message));
+    },
+  },
+  created(): void {
+    this.fetchConfigurationAnchor();
+    this.fetchConfiguredTimestampingServiced();
+    this.fetchApprovedCertificateAuthorities();
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+  @import "../../../assets/colors";
+  @import '../../../assets/tables';
+
+  h3 {
+    color: $XRoad-Grey60;
```

Comment:
The hearers are "black" or darker in the design. 
Also some fonts seem larger or bolder compared to the design.

Reply:
Fixed the styling to better match the design